### PR TITLE
Fix OIDC gitiea redirect URL

### PR DESCRIPTION
**Pull request by Bubu**
See https://github.com/matrix-org/synapse/pull/9288 but wasn't updated here.

Signed-off-by: Marcus Hoffmann <bubu@bubu1.eu>

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
* [x] Pull request includes a [sign off](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#sign-off)
* [x] Code style is correct (run the [linters](https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md#code-style))

### CODE REVIEW

**Comment by Bubu** [comment]
(quickly done from the webUI, I can clean this up some time next week, if nobody gets around to just merging it beforehand)

**Comment by clokep** [comment]
Looks like #9288 and #9134 both got based on the same thing, so the merging didn't go quite as planned. 😄 

Could you sign off on this please?

**Comment by Bubu** [comment]
Added the SOB to the PR description, hope that is enough. Also seems like creating a newsfile from github web UI is tricky.

**Comment by clokep** [comment]
> Also seems like creating a newsfile from github web UI is tricky.

Yeah this is a bit annoying, you can do it by going to the folder (https://github.com/Bubu/synapse/tree/patch-3/changelog.d) and then clicking "Add file"

**Comment by Bubu** [comment]
Hopefully all good now?

File: changelog.d/9404.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (clokep) [inline changelog.d/9404.doc:1]:
```suggestion
Update docs for using Gitea as OpenID provider.
```

**Comment by Bubu** [comment]
Huh... Idk what happened on the first try. Hopefully done now.

**Review by clokep — APPROVED** [review]

**Comment by clokep** [comment]
Thanks!

**Pull request merged**